modules/exploits/linux/local/dirtycow.rb

```diff
@@ -0,0 +1,184 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GreatRanking
```

Comment:
This is still unaddressed: https://github.com/rapid7/metasploit-framework/pull/7476#discussion_r84598813.